controller/volume_controller.go

```diff
@@ -775,7 +779,7 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 
 	if len(rs) == 0 {
 		// first time creation
-		if err = vc.replenishReplicas(v, e, rs); err != nil {
+		if err = vc.replenishReplicas(v, e, rs, ""); err != nil {
```

Comment:
Why we're using `err = ` instead of `err :=` here?

Reply:
Thank you! Fixed